src/libdrakvuf/drakvuf.c

```diff
@@ -482,6 +482,16 @@ os_t drakvuf_get_os_type(drakvuf_t drakvuf)
     return drakvuf->os;
 }
 
+page_mode_t drakvuf_get_page_mode(drakvuf_t drakvuf)
```

Comment:
Are these really needed? There is already API on LibVMI to get these.

Reply:
* For consistency with drakvuf_get_os_type()
* More layered API
* LibVMI does some calculatios on each call...